sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -19,6 +19,19 @@ export class EducationProgramApi extends HttpBaseClient {
     }
   }
 
+  public async getInstitutionForProgram(): Promise<any> {
```

Comment:
we can avoid any and use an interface

Reply:
@ann-aot  for this case, we used any for a purpose, as we do not know if we need more variables from the institution object related to program.

Reply:
on the API side, I see you are returning a DTO promise, 
![image](https://user-images.githubusercontent.com/77353155/132754724-d868d71c-3f79-4b69-9c32-f01721720d74.png)
let me know if I am missing something

Reply:
yeah, its just on the DTO level. It contains only the institutionType. So in the future, if we need more values, we can expand this DTO and the just the API change is enough.

![image](https://user-images.githubusercontent.com/62901416/132755140-332a7f22-9c42-4d6f-8605-6555a49c8bb9.png)